Pipfile

```diff
@@ -4,10 +4,10 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-sceptre = "2.5.0"
+sceptre = "*"
```

Comment:
shouldn't this be set to the same version as in the GH action workflow?

Reply:
oh, yeah it should, thanks